### PR TITLE
Message in background importer when user tries to import more files that is allowed

### LINKED ISSUE #2856: Design call to action for a user to upgrade when try to connect more than one dataset at the same time

### PR DESCRIPTION
**Pull request by xavijam**
So, if the user tries to import at the same time, more files than allowed, those messages will appear:

- If user can upgrade his/her account:
screen shot 2015-04-06 at 16 42 48

- If user can't upgrade (already paid or open source):
screen shot 2015-04-06 at 16 42 55
(* Number is irrelevant :D)

Related things:
- If limits view appears meanwhile an import is happening, and it finishes, we won't redirect to the dataset/map if that view is there.
- If limits view is there, and old imports have finished, if you try to import a new one, it will let you and the limit view will disappear.

Fixes #2856.

Reviewer: @javierarce, please.

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2786/))

**Comment by xavijam** [comment]
ping @javierarce

File: lib/assets/javascripts/cartodb/new_dashboard/views/background_importer/background_importer_limit_view.jst.ejs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="ImportItem-text is-long">
+  <% if (isUpgradeable) { %>
+    In a hurry? <a href="<%= upgradeUrl %>">Upgrade your account</a> to import more files at a time
+  <% } else { %>
+    Unfortunally you can import up to <%= importQuota %> at the same time
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/new_dashboard/views/background_importer/background_importer_limit_view.jst.ejs]: I'd change the message with: "Unfortunally you can only import up to <%= importQuota %> files at the same time"

File: lib/assets/javascripts/cartodb/new_dashboard/views/background_importer/background_importer_limit_view.jst.ejs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="ImportItem-text is-long">
+  <% if (isUpgradeable) { %>
+    In a hurry? <a href="<%= upgradeUrl %>">Upgrade your account</a> to import more files at a time
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/new_dashboard/views/background_importer/background_importer_limit_view.jst.ejs]: I would say "to import several files at a time".

**Comment by javierarce** [comment]
:dog: Code looks good. I added a couple of suggestions related to the texts.

**Comment by xavijam** [comment]
Thanks man :corn:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2815/))

**Pull request merged**